README.md

```diff
@@ -193,6 +195,25 @@ data.current # => OpenWeather::Models::OneCall::CurrentWeather
 data.hourly # => Array[OpenWeather::Models::OneCall::HourlyWeather]
 ```
 
+### Stations
+
+The [Stations API](https://openweathermap.org/stations) lets your manage personal weather stations and measurements.
+
+#### Register a Station
+
+To register a station, you can call the client method:
+```ruby
+data = client.register_station(external_id: 'SF_TEST001', ...) # => OpenWeather::Models::Station
+data.id # => '5ed2118acca8ce0001f1aeg1'
+data.external_id # => 'SF_TEST001'
+```
+Alternatively, call `register!` on the `Station` class:
+```ruby
+data = OpenWeather::Models::Station.register!(external_id: 'SF_TEST001', ...)
```

Comment:
Feels like if we're doing OOP, this should use the model data, e.g.:

```ruby
station = OpenWeather::Models::Station.new(external_id: ...)
station.register!
```

Reply:
Hmmm i tried this, but what felt a little weird was that the client returns another instance of Station. I could disregard that return I suppose, mutating and returning the caller. Does that sound good to you?

This happened to feel more Rail-sy as well, though that’s not a strong argument :D

Reply:
I would assign the result to the attributes of the instance and return `self`.